sdk/core/core-http/src/serializer.ts

```diff
@@ -697,21 +697,28 @@ function deserializeCompositeType(
         );
       } else {
         const propertyName = xmlElementName || xmlName || serializedName;
-        let unwrappedProperty = responseBody[propertyName!];
         if (propertyMapper.xmlIsWrapped) {
-          unwrappedProperty = responseBody[xmlName!];
-          unwrappedProperty = unwrappedProperty && unwrappedProperty[xmlElementName!];
-
-          const isEmptyWrappedList = unwrappedProperty === undefined;
-          if (isEmptyWrappedList) {
-            unwrappedProperty = [];
-          }
+          /* a list of <xmlElementName> wrapped by <xmlName>
+            For the xml example below
+              <Cors>
+                <CorsRule>...</CorsRule>
+                <CorsRule>...</CorsRule>
+              </Cors>
+            the responseBody has
+              {
+                Cors: {
+                  CorsRule: [{...}, {...}]
+                }
+              }
+            xmlName is "Cors" and xmlElementName is"CorsRule".
+          */
+          const wrapped = responseBody[xmlName!];
+          const elementList = wrapped ? wrapped[xmlElementName!] : [];
```

Comment:
out of curiosity could this be written like

```ts
const elementList = wrapped?.[xmlElementName!] ?? [];
```

Reply:
yes it worked. Updated to use ?. and ??